raft.go

```diff
@@ -602,17 +632,24 @@ func (r *Raft) Stats() map[string]string {
 	lastLogIndex, lastLogTerm := r.getLastLog()
 	lastSnapIndex, lastSnapTerm := r.getLastSnapshot()
 	s := map[string]string{
-		"state":                r.getState().String(),
-		"term":                 toString(r.getCurrentTerm()),
-		"last_log_index":       toString(lastLogIndex),
-		"last_log_term":        toString(lastLogTerm),
-		"commit_index":         toString(r.getCommitIndex()),
-		"applied_index":        toString(r.getLastApplied()),
-		"fsm_pending":          toString(uint64(len(r.fsmCommitCh))),
-		"last_snapshot_index":  toString(lastSnapIndex),
-		"last_snapshot_term":   toString(lastSnapTerm),
-		"latest_configuration": fmt.Sprintf("%+v", r.configurations.latest),
+		"state":               r.getState().String(),
+		"term":                toString(r.getCurrentTerm()),
+		"last_log_index":      toString(lastLogIndex),
+		"last_log_term":       toString(lastLogTerm),
+		"commit_index":        toString(r.getCommitIndex()),
+		"applied_index":       toString(r.getLastApplied()),
+		"fsm_pending":         toString(uint64(len(r.fsmCommitCh))),
+		"last_snapshot_index": toString(lastSnapIndex),
+		"last_snapshot_term":  toString(lastSnapTerm),
+	}
+
+	configuration, _, err := r.GetConfiguration()
```

Comment:
oops, didn't realize this was an external call before. I'm starting to think we should move all these application-level API calls into a separate file, perhaps called api.go.